Before: a75b7e5c2cf3b7a1458fc55ea89f0883c821ed0b4fcb8d655d0a13d98f9cb0ae
After: 33bfbc981a0607f4ba3019729ce32d3a0171520c560873cc5a84d42f841bc3e9
Patch: a584450ce434eec012f683053334fcb093e65dfad69010590e9f34d694b4abd4
--- corvid-features/plugin.rb	2012-08-13 19:24:43.319622766 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,7 +0,0 @@
-install {
-  empty_directory 'resources'
-  empty_directory 'resources/latest'
-  copy_file       'resources/.gitignore'
-  install_feature 'corvid', 'test_spec', run_bundle: true, say_if_installed: false
-  copy_file       'test/spec/resources_spec.rb'
-}
--- lib/corvid/%name%_feature.rb.tt	2012-08-13 16:20:51.157407448 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,10 +0,0 @@
-require 'corvid/feature'
-
-class <%= class_name %> < Corvid::Feature
-
-  since_ver <%= since_ver %>
-
-  rake_tasks {
-  }
-
-end
--- lib/corvid/%name%_plugin.rb.tt	2012-08-13 12:56:43.053579014 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,10 +0,0 @@
-require 'corvid/plugin'
-
-class <%= class_name %> < Corvid::Plugin
-
-  require_path '<%= require_path %>'
-
-  feature_manifest ({
-  })
-
-end
--- resources/.gitignore	2012-08-09 13:34:14.000844233 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1 +0,0 @@
-latest
--- resources/latest/corvid-features/%name%.rb.tt	2012-08-13 14:12:45.145097126 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,13 +0,0 @@
-install {
-  # empty_directory 'lib'
-  # copy_file       'hello.txt'
-  # copy_executable 'bin/hello'
-}
-
-# update {|ver|
-#   case ver
-#   when 2
-#     update stuff here
-#     empty_directory 'lib'
-#   end
-# }
--- test/spec/%name%_plugin_spec.rb.tt	2012-08-13 19:37:01.847395107 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,11 +0,0 @@
-# encoding: utf-8
-require_relative '../bootstrap/spec'
-require 'corvid/test/resource_patch_tests'
-require '<%= require_path %>'
-
-describe <%= class_name %> do
-  include Corvid::ResourcePatchTests
-  res_patch_dir "#{APP_ROOT}/resources"
-
-  include_feature_update_install_tests '<%= name %>', <%= class_name %>.new
-end
--- test/spec/resources_spec.rb	2012-08-11 12:03:08.949488981 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,10 +0,0 @@
-# encoding: utf-8
-require_relative '../bootstrap/spec'
-require 'corvid/test/resource_patch_tests'
-
-describe "Actual Resources" do
-  include Corvid::ResourcePatchTests
-  res_patch_dir "#{APP_ROOT}/resources"
-
-  include_patch_validity_tests
-end
